packages/widgets/bettinggame/src/main.js

```diff
@@ -0,0 +1,63 @@
+import {appFactory, selection} from 'tocco-app-extensions'
+import {reducer as reducerUtil, appContext} from 'tocco-util'
+
+import TournamentWidget from './components/TournamentWidget'
+import reducers, {sagas} from './modules/reducers'
+
+const packageName = 'bettinggame'
+
+const initApp = (id, input, events, publicPath) => {
+  const content = <TournamentWidget formBase={input.formBase} tournamentKey={input.tournamentKey} />
```

Comment:
suggestion(blocking): so that the components run in the correct environment the `env` has to be set correctly.

https://docs.tocco.ch/framework/configuration/widgets/widget.html#env 
- No footer information should be visible 
![image](https://user-images.githubusercontent.com/6946713/199259298-c87106ce-8e71-4c00-934e-299f5a38164b.png)
- correct theme has to be applied